src/edfi-paging-test/edfi_paging_test/api/paginated_result.py

```diff
@@ -0,0 +1,94 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+from typing import Any, Dict, List, Optional
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from edfi_paging_test.api.request_client import RequestClient
+
+
+class PaginatedResult():
+    """
+    The PaginatedResult class is bound with information from the response of the
+    api, specifically when it returns a list of items that could be paginated.
+    Parameters
```

Comment:
Easier to read with some additional line breaks.

```suggestion

    Parameters
```